target/linux/ipq806x/patches-5.4/095-ipq806x-phy-qualcomm-add-qcom-ipq806x-dwc-usb-phy-driver.patch

```diff
@@ -0,0 +1,632 @@
+From ef19b117b83466e1c030368101a24367a34be7f0 Mon Sep 17 00:00:00 2001
+From: Ansuel Smith <ansuelsmth@gmail.com>
+Date: Fri, 17 Jul 2020 15:16:31 +0200
+Subject: phy: qualcomm: add qcom ipq806x dwc usb phy driver
+
+This has lost in the original push for the dwc3 qcom driver.
+This is needed for ipq806x SoC as without this the usb ports
+doesn't work at all.
+
+Signed-off-by: Andy Gross <agross@codeaurora.org>
+Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>
+Tested-by: Jonathan McDowell <noodles@earth.li>
+Link: https://lore.kernel.org/r/20200717131635.11076-1-ansuelsmth@gmail.com
+Signed-off-by: Vinod Koul <vkoul@kernel.org>
+---
+
+Combined with 3d7b0ca5300bd01b176f2b4c10e173db802560d8
+Light modification to Kconfig as some config are missing in kernel 5.4
```

Comment:
The patch doesn't apply without this modification to kconfig and makefile... Can i leave this?

Reply:
The modifications are fine. Of course, backporting is necessary, and we cannot just add a patch from a newer kernel as-is in any case :-)